pkg/policy/execute.go

```diff
@@ -175,7 +167,7 @@ func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Pol
 		qr, err := e.executeQuery(ctx, q)
 		if err != nil {
 			e.log.Error("failed to execute query", "err", err)
-			return nil, fmt.Errorf("%s/%w", policy.Name, err)
+			return nil, diag.FromError(fmt.Errorf("%s/%w", policy.Name, err), diag.DATABASE)
```

Comment:
```suggestion
			return nil, diag.FromError(fmt.Errorf("%s: %w", policy.Name, err), diag.DATABASE)
```

This `%s/%w` syntax isn't good, let's make it the Go way which is `<related object>: <error about object>`